### PR TITLE
Migrate Metrics query editor 'form-control' to 'pf-c-form-control'

### PR DESCRIPTION
**Pull request by rhamilto**
Supports https://jira.coreos.com/browse/CONSOLE-1564

### CODE REVIEW

File: frontend/public/components/monitoring/_monitoring.scss [hunk 1]
```diff
@@ -72,6 +72,7 @@
 }
 
 .query-browser__query {
+  height: 33px !important;
```
Comment (spadgett) [inline frontend/public/components/monitoring/_monitoring.scss]: Does shift-enter in the field work properly with this change?

Comment (rhamilto) [reply frontend/public/components/monitoring/_monitoring.scss]: Good catch!  Yeah, but not like it did before.  Lemme rethink.

Comment (rhamilto) [reply frontend/public/components/monitoring/_monitoring.scss]: Fixed with a better solution.  Thanks, @spadgett.

**Pull request merged**